docs/documentation/_data/sidebars/main.yml

```diff
@@ -49,6 +49,8 @@ entries:
                       url: /modules/020-deckhouse/configuration.html
                     - title: Custom Resources
                       url: /modules/020-deckhouse/cr.html
+                - title: FAQ
+                  url: /modules/020-deckhouse/faq.html
```

Comment:
As for me, it is more about the control plane manager / node manager. Although, the deckhouse module is also ok.

Reply:
It was my main concern. But I couldn't decide between plane manager and node manager, so I preferred to move it in deckhouse module)